src/helpers/Sign/crypto/scrypt-rom.js

```diff
@@ -0,0 +1,149 @@
+
+function blockxor(S, Si, D, Di, len){
```

Comment:
is there any pre-existing library we can use here instead of having the code explicitly implemented?